microsite/core/Footer.js

```diff
@@ -1,4 +1,6 @@
 /*
+* Made with <3 at Spotify
+
```

Comment:
This is in the license header which must not be changed, and the result won't be visible to the user since it's a comment.

Reply:
We need to add this in the same way that it appears here: https://klio.io/

Where can I add it so it'll appear?

Reply:
Klio is owned by Spotify, but Backstage is not, so I don't think we add that https://backstage.io